docs/guide/design-principles/redundancy.md

```diff
@@ -63,6 +65,12 @@ To achieve this, disable the primary endpoint after failover. Note that if the m
 
 **Include redundancy for your routing solution**. Consider designing a [Global routing redundancy solution](../networking/global-web-applications/overview.md) for mission-critical web applications.
 
+**Test and validate your redundant components**. Adding redundancy increases complexity which in turn reduces availability. To ensure that adding redundancy actually leads to higher availability, you should test and validate followings: 
```

Comment:
Is this really a "multi-region" bullet?  I don't see anything about region in here. Should this instead be the last bullet point in the Recommendations H2 instead of the last bullet point in the Multi-region H3?

Reply:
Relocated it